Header header
float64 exposure
float64 gain 